GCPAuthTokens/token.go

```diff
@@ -0,0 +1,79 @@
+package pkg
+
+import (
+	"context"
+	"os"
+
+	credentials "cloud.google.com/go/iam/credentials/apiv1"
+	"google.golang.org/api/option"
+	credentialsPb "google.golang.org/genproto/googleapis/iam/credentials/v1"
+)
+
+const (
+	EnvServiceAcctFile = "GOOGLE_APPLICATION_CREDENTIALS"
+)
+
+func isEnvExist(key string) bool {
+	if _, ok := os.LookupEnv(key); ok {
+		return true
+	}
+	return false
+}
```

Comment:
I don't think we have to branch here. `ok` is already a boolean so, we can return it directly.

But I don't think we need this function.
We can directly use `os.Getenv` or `os.LookupEnv` and do the branching directly there.

Reply:
Good call
Fixed now